CMakeLists.txt

```diff
@@ -633,11 +633,20 @@ ENDIF()
 SET(CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS_DEBUG} -DLMMS_DEBUG")
 SET(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -DLMMS_DEBUG")
```

Comment:
From what I can tell, both of these are unused now and should be removed.

Reply:
Nah it ain't, the macro is still used to enable debug printing to the console, in other places.

Reply:
Gotcha.